be-nodejs/src/scores/scores.controller.spec.ts

```diff
@@ -1,33 +1,198 @@
-describe('Score Calculator', () => {
+import { stringLiteral } from "@babel/types";
 
-    function ScoreCalculator(){
-        
-        //[1-9]
+function ParseRoll(roll: string){
+    const number = parseInt(roll);
+    
+    if(number || number === 0){
+        return number;
+    }
+    
+    if(roll === 'x'){
```

Comment:
I believe all of our api input data has been validated to include 'X's for strikes, not 'x'